Source/Core/Gen3/Searchers/StaticSearcher3.cpp

```diff
@@ -66,11 +66,24 @@ void StaticSearcher3::startSearch(const std::array<u8, 6> &min, const std::array
     }
 }
 
+void buggedRoamerIVsMatcher(std::array<u8, 6> &ivs, const bool buggedRoamer)
```

Comment:
Unless I'm misunderstanding, this essentially does what colo does. The IVs set but the function aren't passed to the LCRNG reverse function. These are only used for the results.

Reply:
Oh right I might to leave this comment in the other spot.

I still prefer the way I had it where it is a simple if statement instead of a function call

```
std::array<u8, 6> ivs;
if (staticTemplate->getRoamer())
{
    ivs = { hp, atk & 7, 0, 0, 0, 0 };
}
else
{
    ivs = { hp, atk, def, spa, spd, spe };
}
```

Reply:
Done